nixos/modules/config/gtk/gtk.nix

```diff
@@ -0,0 +1,146 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.gtk;
+  gtk2 = cfg.enable && cfg.gtk2;
+
+  toGtk2Ini = key: value:
+    let
+      value' =
+        if isBool value then (if value then "true" else "false")
+        else if isString value then "\"${value}\""
+        else toString value;
+    in
+      "${key} = ${value'}";
+  toGtk3Ini = generators.toINI {
```

Comment:
GTK3 uses GKeyFile, not ini: https://gitlab.gnome.org/GNOME/gtk/blob/3.24.12/gtk/gtksettings.c#L75-86

Reply:
@jtojnar Renamed. If you're about implementation, it was borrowed from @rycee https://github.com/rycee/home-manager/blob/master/modules/misc/gtk.nix `home-manager` gtk module and works fine for people